plans/views.py

```diff
@@ -400,16 +401,21 @@ def get_redirect_url(self, **kwargs):
             return redirect_url + "?next=" + next_url
 
 
+class NextUrlMixin(object):
```

Comment:
1. I don't think you need to inherit from `object` anymore, unless you have an explicit reason ?
2. We need some good documentation over this mixin class, why it's being used and why it matters. 
3. Maybe check the `next` has the same `HOST` as the application and if not reject it ? _(I think Django auth login does the same thing when it comes to verifying the `next` param)_

Reply:
1. Fixed
2. I have added some comment about this.
3. I think, that current code is OK, because it uses only the path part. But I will take closer look at the Django auth implementation.

Reply:
@Alir3z4 I have looked into the Django implementation and reworked it into [`django-next-url-mixin`](https://github.com/PetrDlouhy/django-next-url-mixin) application with tests and everything. The behaviour is a bit different, than the original implementation here - if the url is not safe, the next parameter is not taken into account.

If you would like to have this mixin internally, I can copy the code from the application, but I think, that it is better to use the app without code duplication.